test/CadlRanchProjects/inheritance/Generated/Configuration.json

```diff
@@ -5,5 +5,6 @@
   "SharedSourceFolders": [
     "../../../../artifacts/bin/AutoRest.CSharp/Debug/net6.0/Generator.Shared",
     "../../../../artifacts/bin/AutoRest.CSharp/Debug/net6.0/Azure.Core.Shared"
-  ]
+  ],
+  "unreferenced-types-handling": "keepAll"
```

Comment:
Why we have this change in configuration? Do we need this?

Reply:
I was told that this `inheritance` project is not a cadl ranch project, therefore I should not add this config in its cadl-project.yaml. I will remove this.

Reply:
reverted